### ISSUE TITLE
Heading hierarchy on develop your curriculum needs to follow a logical order

### ISSUE THREAD

**Issue by jack-skerrett-oak** [oaknational/Oak-Web-Application#1363]
The getting started/how to section is currently a little funky.
No page should skip a heading level and they should be used to describe the page layout

**Acceptance Criteria:**
- I think "Getting started" should stay as a h3
- "What element of curriculum design..." should stay as a h4
- "How to design a unit of study" and so on should all be h5s

**Comment by jack-skerrett-oak** [comment]
https://www.notion.so/oaknationalacademy/The-heading-hierarchy-for-this-section-on-the-Develop-your-curriculum-page-does-not-follow-a-logic-6ccc6e9239be402d8357d82c0612da3e?pvs=4
